sql/parse/parse.go

```diff
@@ -442,6 +442,7 @@ func convertShow(ctx *sql.Context, s *sqlparser.Show, query string) (sql.Node, e
 					filter = expression.NewLike(
 						expression.NewUnresolvedColumn("Table"),
 						expression.NewLiteral(s.ShowTablesOpt.Filter.Like, sql.LongText),
+						nil,
```

Comment:
Is it possible to use ESCAPE on these LIKE expressions? If so add a TODO

Reply:
It does seem like it is possible to use ESCAPE for SHOW ... LIKE ... statements.
MySQL throws a Syntax Error.